x/storage/keeper/keeper.go

```diff
@@ -915,6 +933,11 @@ func (k Keeper) CancelCreateObject(
 
 	bbz := k.cdc.MustMarshal(bucketInfo)
 	store.Set(types.GetBucketByIDKey(bucketInfo.Id), bbz)
+	if ctx.IsUpgraded(upgradetypes.Pawnee) {
+		if objectInfo.ObjectStatus == types.OBJECT_STATUS_CREATED {
+			k.DecreaseCreatedObjectCount(ctx, bucketInfo.Id)
```

Comment:
also need to `DecreaseCreatedObjectCount` when `rejectSeal`

Reply:
sure